### ISSUE TITLE
Include nightly toolchain date in sort

### ISSUE THREAD

**Issue by dtolnay** [gsquire/toolchain_find#1]
Nightly version strings look like:

```console
rustc 1.33.0-nightly (01f8e25b1 2019-01-24)
```

**Comment by gsquire** [comment]
I suppose I could also grab the date when I parse this version string and if I see two nightly toolchains, use that as a second comparison. I am curious though. Is it possible to have more than one nightly toolchain on a machine?

**Comment by dtolnay** [comment]
Yes, I often have dozens installed. :D

```console
$ rustup install nightly-2019-01-01
$ rustup install nightly-2019-01-02
$ rustup install nightly-2019-01-03
$ rustup install nightly-2019-01-04
$ rustup install nightly-2019-01-05
$ rustup install nightly-2019-01-06
$ rustup install nightly-2019-01-07
$ rustup install nightly-2019-01-08
$ rustup install nightly-2019-01-09
$ rustup install nightly-2019-01-10
```

**Comment by gsquire** [comment]
I just published [version 0.1.1](https://crates.io/crates/toolchain_find/0.1.1). Let me know if you still run into any issues.
